crates/primitives/class/src/compile.rs

```diff
@@ -134,9 +133,15 @@ fn parse_sierra_version(program: &[Felt]) -> Result<SierraVersion, ClassCompilat
         [first, ..] if first == &VERSION_0_1_0_AS_SHORTSTRING => Ok(SierraVersion(0, 1, 0)),
         [a, b, c, ..] => {
             let (a, b, c) = (
-                felt_to_u64(a).map_err(|_| ClassCompilationError::ParsingSierraVersion("malformed version".into()))?,
-                felt_to_u64(b).map_err(|_| ClassCompilationError::ParsingSierraVersion("malformed version".into()))?,
-                felt_to_u64(c).map_err(|_| ClassCompilationError::ParsingSierraVersion("malformed version".into()))?,
+                a.to_owned()
```

Comment:
this to_owned makes the &Felt -> Felt conversion right? maybe change to .clone() because that was making me confused

Reply:
done
replaced `to_owned` with dereferencing